apache/zeppelin
# [ZEPPELIN-908] Apply new mechanism to CassandraInterpreter

### What is this PR for?
This handles replacing the registration of interpreter with static block by the interpreter-setting.json file

### What type of PR is it?
Sub-ask

### Todos
NA

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-908

### How should this be tested?
* There shouldn't be any warning like below on starting the server

`INFO [2016-09-29 00:25:46,247] ({main} CassandraInterpreter.java[<clinit>]:155) - Bootstrapping Cassandra Interpreter
 WARN [2016-09-29 00:25:46,250] ({main} Interpreter.java[register]:347) - Static initialization is deprecated for interpreter cassandra, You should change it to use interpreter-setting.json in your jar or interpreter/{interpreter}/interpreter-setting.json
 INFO [2016-09-29 00:25:46,250] ({main} InterpreterFactory.java[init]:204) - Interpreter cassandra.cassandra found. class=org.apache.zeppelin.cassandra.CassandraInterpreter`

* And ensure that the cassandra related paragraphs run without any error

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
